guides/doc-Monitoring_Project/topics/proc_retrieving-archived-metrics-using-cli.adoc

```diff
@@ -0,0 +1,59 @@
+[id='retrieving-archived-metrics-cli_{context}']
+= Retrieving Archived Metrics using CLI
+
+You can use the PCP CLI tools to retrieve metrics from an archive file. To do that, add the `--archive` parameter and specify the archive file.
```

Comment:
```suggestion
You can use the PCP CLI tools to retrieve metrics from an archive file. 
To do that, add the `--archive` parameter and specify the archive file.
```
I am doing this for each one so that you can directly apply from here (WebUI).